cmdx.py

```diff
@@ -2684,6 +2768,9 @@ def type(self):
 
         return self._mplug.attribute().apiTypeStr
 
+    # typeName is used as key for ObjectSet.sort
+    typeName = None
+
```

Comment:
What's going on here? :O

Reply:
When i tried to use ObjectSet.sort() it crashed when i had components or plugs in it. So i had to had a typeName variable for them just to make the .sort work

This is probably not the best approach. I'll have to rewrite .sort to avoid adding that